scripts/dts/edtlib.py

```diff
@@ -184,6 +185,12 @@ def __init__(self, dts, bindings_dirs, warn_file=None,
           If True, set the Node.bus for 'fixed-partitions' compatible nodes
           to None.  This allows 'fixed-partitions' binding to match regardless
           of the bus the 'fixed-partition' is under.
+
+        infer_binding_for_paths (default: None):
+          An iterable of devicetree paths identifying nodes for which bindings
+          should be inferred from the node content.  Pass none if no nodes
+          should support inferred bindings.
```

Comment:
Can you clarify in here if child of the path are covered or just the path itself?

Reply:
(probably add to the docs as well).

Reply:
OK, but it does say "path", not "path prefix".